libs/libc/stdio/lib_remove.c

```diff
@@ -56,7 +56,7 @@ int remove(FAR const char *path)
    */
 
   if (unlink(path) != 0  /* If it is indeed a directory...  */
```

Comment:
Should it be
```suggestion
  if (unlink(path) != 0        /* If it is indeed a directory...  */
```
now?

Reply:
Done.